care/facility/api/viewsets/patient.py

```diff
@@ -2,11 +2,12 @@
 import json
 from json import JSONDecodeError
 from config.celery_app import app
-
+from django.conf import settings
 from django.contrib.postgres.search import TrigramSimilarity
 from django.core.validators import validate_email
 from django.db.models.query_utils import Q
 from django_filters import rest_framework as filters
+from djqscsv import render_to_csv_response
```

Comment:
Codacy found an issue: [Unable to import 'djqscsv'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5753282)